doc/harmony_ap_clustering_user_guide.md

```diff
@@ -37,15 +37,20 @@ This document is licensed under the Creative Commons Attribution-ShareAlike 4.0
   * [3.1 Docker Compose configuration](#31-docker-compose-configuration)
   * [3.2 ActiveMQ configuration](#32-activemq-configuration)
   * [3.3 Nginx configuration](#33-nginx-configuration)
+  * [3.4 Running the environment](#34-running-the-environment)
 
 <!-- vim-markdown-toc -->
 
 ## 1 Introduction
 
-Harmony eDelivery Access Access Point offers the ability of being deployed in a cluster. This configuration will provide high availability capabilities, so it can operate continuously without intervention if some of the cluster components fail.
+Harmony eDelivery Access Point offers the ability of being deployed in a cluster. This configuration will provide high availability capabilities, so it can operate continuously without intervention if some of the cluster components fail.
 
 The high availability is achieved by deploying multiple instances of the Access Point in a cluster. The Access Point instances are configured to use the same message broker subclusters so that all the services involved in the setup have a fallback instance.
 
+The following diagram shows a clustered setup of the Access Point:
+
+![clustered setup](img/ug_ap_cluster_overview_diagram.svg)
```

Comment:
Overall, the diagram looks good. However, I think it would be good to clarify the security boundaries of different endpoints of the Access Point. The Admin UI and the `/services/wsplugin` endpoint should be accessed from internal network only while the `/services/msh` endpoint is accessed by other Access Points and therefore, it's accessed from external networks. For example, the "Other services" box could be splitted into "Backend clients" and "Other Access Points". Also, the source network (internal / external) could be included in the diagram. The AP [installation guide](https://github.com/nordic-institute/harmony-common/blob/develop/doc/harmony-ap_installation_guide.md#22-network-diagram) can be referenced for more detailed information about recommended firewall configuration.

In addition, "Distributed database" should probably be "Shared database"?